build.psm1

```diff
@@ -848,6 +848,17 @@ function Start-PSPester {
         [switch]$IncludeFailingTest
     )
 
+    if (-not (Test-Path $Pester)) {
+        Write-Warning @"
+Pester module not found. Make sure that the proper git submodules are installed by running:
+
+git submodule update --init 
+
+
+"@
```

Comment:
A minor comment about the message -- how about this:
```
Write-Warning @"
Pester module not found.
Make sure that the proper git submodules are installed by running:
    git submodule update --init
"@
```
It looks like this:
![image](https://user-images.githubusercontent.com/127450/31364498-f1afb6ca-ad19-11e7-92fa-869e3de3d25a.png)